### DISCUSSION TITLE
'MATEKF405***TE***'

### DISCUSSION THREAD

**Discussion by FPVmyRoom** [iNavFlight/inav-configurator#1542] [Q&A] [answered]
Hello, when I want to flash my MATEKSys F405TE MiniTE Flight Controller with the newest INAV 5.0.0 Configurator, which is described as to now feature this spcific board, I can only find the MATEKF405 Target, not the one for the TE. 

Anyone know how to get the flash for this TE mini FlightController? 
#

Thanks and Greetz

**Comment by MrD-RC** [comment, marked as answer]
iNav 5.0 is currently in pre-released. I would suggest waiting until it is released as stable before using it.

**Reply by supernova1969a** [reply]
I have a Iflight Evoque. With INAV 5.0 . I am trying to get the LEDs working again. The line Resource LED_STRIP 1 A08 is in the original betaflight dump...how can I add this to INAV 5.0. I tried CLI..got errors. thanks so much!
